service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2293,6 +2337,89 @@ components:
         round-trip:
           type: string
 
+    ExportRequest:
+      type: object
+      properties:
+        exportModel:
+          description: Name of the export model to use
+          type: string
+        inputs:
+          description: Key-value map of the export model inputs (e.g. filenamePrefix=test)
+          type: object
+          additionalProperties:
+            type: string
+            nullable: true
+        redirectBackUrl:
+          type: string
+          nullable: true
+        study:
+          type: string
+          nullable: false
+        cohorts:
+          description: List of cohort ids
+          type: array
+          items:
+            type: string
+        includeAnnotations:
+          description: Include the annotation data when true
+          type: boolean
+        instanceQuerys:
+          # TODO: Replace this with a list of concept set ids once we can build the entity filters on the backend.
+          type: array
+          items:
+            type: object
+            properties:
+              entity:
+                type: string
+                nullable: false
+              query:
+                $ref: "#/components/schemas/QueryV2"
+
+    ExportResult:
+      type: object
+      properties:
+        status:
+          type: string
+          enum: [ "RUNNING", "COMPLETE" ]
```

Comment:
What does RUNNING mean?

Reply:
I was thinking we'll have export options that just kick something off (e.g. AoU genomic extraction job).

Reply:
I was thinking we'll have export options that just kick something off (e.g. AoU genomic extraction job).